### DISCUSSION TITLE
Couldn't find anything to add shadow/ halo around pathlayer

### DISCUSSION THREAD

**Discussion by khyati-k** [visgl/deck.gl#5582] [Q&A]
Hi everyone, I have been trying to add a shadow/halo around pathlayer, but couldn't find any properties or extensions for that.
I did try to use lighting api but it was only for polygons as per documentation. Am I missing something or this feature isn't available.

**Comment by AdriSolid** [comment]
This could be useful: https://codepen.io/Pessimistress/pen/MWyaWxV?editors=1010

An example extending `PathLayer` with some stroke properties.
